### ISSUE TITLE
License inconsistency about the Connectors Bundle Docker

### ISSUE THREAD

**Issue by michaelarnauts** [camunda/connectors#1640]
__Describe the Bug__

There is an inconsistency about the License to use the Camunda Connectors Bundle Docker image.

----

On this page: https://docs.camunda.io/docs/reference/licenses/#connectors
> The Connector SDK, REST Connector, Connector Runtime Docker image, and Connectors Bundle Docker image are licensed under the [Apache 2.0 license](https://www.apache.org/licenses/LICENSE-2.0).

It seems that the Connectors Bundle Docker image is available under the Apache 2.0 license, so freely available to use in production.

----

On this page: https://camunda.com/blog/2022/05/how-open-is-camunda-platform-8/
> The short summary is that you can run everything green (including green stripes) as [self-managed](https://docs.camunda.io/docs/self-managed/overview/) in production without needing a license.

It seems to indicate that the Connectors are available to use together with the Zeebe engine withouth a license. (It's not marked as blue), so freely available to use in production.

----

On this page: https://github.com/camunda/connectors#modules-available-under-camunda-self-managed-free-edition-license

> Modules available under [Camunda Self-Managed Free Edition license](https://camunda.com/legal/terms/cloud-terms-and-conditions/camunda-cloud-self-managed-free-edition-terms/)
>
> * All [Out-of-the-Box Connectors](https://github.com/camunda/connectors/blob/main/connectors) except for REST Connector (see above)
> * [Docker images](https://github.com/camunda/connectors/blob/main/bundle) of the out-of-the-box Connectors for Camunda, bundled with a runtime

It seems the docker image is only available under the Proprietary license, so it can't be used in production without a license.

----

Can this be clarified?

**Comment by sbuettner** [comment]
@crobbins215 I think we need to adapt to the latest license changes right?

**Comment by crobbins215** [comment]
@sbuettner yes I think so, do you have a list of licenses we should be changing to? I wasn't involved in the licence changes. 

@felix-mueller do you have any input in this area to ensure we align with the business? 

@sbuettner I think we can close off https://github.com/camunda/team-connectors/issues/854 and just keep one ticket open.

**Comment by crobbins215** [comment]
Hi, 

Thanks for flagging @michaelarnauts

The Connector SDK, REST Connector, Connector Runtime Docker image, and Connectors Bundle Docker image are licensed under the [Apache 2.0 license](https://www.apache.org/licenses/LICENSE-2.0). The source code of all other Connectors provided by Camunda out of the box is licensed under the [Camunda License v1](https://legal.camunda.com/licensing-and-other-legal-terms#camunda-license).

The compiled software is distributed under a proprietary license, which can be either the [Camunda Self-Managed Non-Production license](https://legal.camunda.com/#self-managed-non-production-terms) or the Camunda Self-Managed Enterprise Edition license (a copy of which you obtain when you contact Camunda). To use the software in production, [purchase the Camunda Self-Managed Enterprise Edition](https://camunda.com/platform/camunda-platform-enterprise-contact/).

**Comment by sbuettner** [comment]
In the context of https://camunda.slack.com/archives/C02JLRNQQ05/p1741274555123489 we might want to switch to a single license for all artifacts.
